pkg/server/internal/testserverclient/server_client.go

```diff
@@ -2258,7 +2258,7 @@ func (cli *TestServerClient) getMetrics(t *testing.T) []byte {
 
 func getStmtCnt(content string) (stmtCnt map[string]int) {
 	stmtCnt = make(map[string]int)
-	r := regexp.MustCompile("tidb_executor_statement_total{db=\"\",type=\"([A-Z|a-z|-]+)\"} (\\d+)")
+	r := regexp.MustCompile("tidb_executor_statement_total{db=\"\",resource_group=\".*\",type=\"([A-Z|a-z|-]+)\"} (\\d+)")
```

Comment:
I think we can remove the `,resource_group=\".*\"`condition, They are equivalent.

Reply:
not requivalent, It's can't match if removed

Reply:
why? it's add a new filter.

Reply:
If not add this filter ,  it returns 0, all the related test will fail,.